java/change-notes/2021-03-05-regex-utils.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* Added models for Apache Commons-Lang's `RegExUtils` class. This means that any query that tracks tainted data may return additional results in cases where a `RegExUtils` transformation is part of the path from source to sink.
```

Comment:
Really minor, but it looks like the project itself does not use a hyphen, see https://commons.apache.org/proper/commons-lang/

```suggestion
* Added models for Apache Commons Lang's `RegExUtils` class. This means that any query that tracks tainted data may return additional results in cases where a `RegExUtils` transformation is part of the path from source to sink.
```